scripts/install.sh

```diff
@@ -68,15 +70,33 @@ fi
 
 DIST="$VERSION"
 
+printf "Do you want the latest release(y/n): "
+read answer
+if [ "$answer" != "${answer#[Yy]}" ] ;then 
+        echo ""; 
+else
+        printf "You can find all the releases here $RELEASE_URL"
+        echo ""
+        printf "Enter relase version (e.g, v0.8.7, 0.8.6): "
```

Comment:
Looks like the `v` suffix is required so include it in the `0.8.6` example.

Reply:
Yeah, I kind of thought the user is gonna copy paste it from github, but prefixed **v** is better

Reply:
Fixed